roles/debian-hardening/vars/main.yml

```diff
@@ -7,11 +7,15 @@ kernel_params:
  - init_on_free=1
  - slab_nomerge
  - pti=on
- - slub_debug=ZF
+ - slub_debug=FZP
  - randomize_kstack_offset=on
  - slab_common.usercopy_fallback=N
  - iommu=pt
  - security=yama
+ - page_poison=on
+ - mce=0
+ - page_alloc.shuffle=1
```

Comment:
Already enabled in the kernel configuration
grep CONFIG_SHUFFLE_PAGE_ALLOCATOR config-6.0.0-0.deb11.6-rt-amd64 
CONFIG_SHUFFLE_PAGE_ALLOCATOR=y